File: vendaval.client/src/app/order/orderStatus.ts
export enum OrderStatus {
  waitingForPayment = 1,
  payed = 2,
  inPreparation = 3,
  shipped = 4,
  outForDelivery = 5,
  delivered = 6,
  cancelRequested = 7,
  canceled = 8,
  waitingForReturnMail = 9,
  returned = 10,
  refundRequested = 11,
  refunded = 12
}

File: Vendaval.Domain/Enums/OrderStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Vendaval.Domain.Enums
{
    public enum OrderStatus
    {
        WaitingForPayment = 1,
        Payed = 2,
        InPreparation = 3,
        Shipped = 4,
        OutForDelivery = 5,
        Delivered = 6,
        CancelRequested = 7,
        Canceled = 8,
        WaitingForReturnMail = 9,
        Returned = 10,
        RefundRequested = 11,
        Refunded = 12
    }
}
